test/compile_errors.zig

```diff
@@ -2,15 +2,14 @@ const tests = @import("tests.zig");
 const builtin = @import("builtin");
 
 pub fn addCases(cases: *tests.CompileErrorContext) void {
-    cases.add("slice sentinel mismatch",
```

Comment:
did you intend to replace this old test case?

Reply:
Good catch, I thought I had copy-pasted it heh